### DISCUSSION TITLE
How do you get air to error out / fail if there is a compilation error?

### DISCUSSION THREAD

**Discussion by phantomdenkali** [air-verse/air#444] [Q&A]
I've been using air in my development. Seems to work well, however, I have found a few times when there is a compliation/build error in the code, air just continues on and runs the last version of the binary somehow? I have to stop air, run go run main.go to find/fix the error and then run air again. Of course, sometimes I don't realize the error and go through my manual testing workflow to find puzzling results since the code doesn't match the runtime

Not sure if I am missing something obvious in my workflow? I use VSCode as my IDE btw

**Comment by andradei** [comment]
You can use `stop_on_error = true`, here's the example:

https://github.com/cosmtrek/air/blob/master/air_example.toml#L44

**Reply by lukmdo** [reply]
+ related 
https://github.com/cosmtrek/air/issues/503

**Comment by lukmdo** [comment]
[`build.stop_on_error`](https://github.com/cosmtrek/air/blob/master/air_example.toml#L44) config/flag should do it

See comments for more.

**Comment by slavanorm** [comment]
can we vote for this config to be default?
it cost me 2 hours of debug yesterday.

**Reply by shinebayar-g** [reply]
Sounds like a reasonable request. :+1: Application keep running when it isn't even compiling is not a good idea.
